lib/test.js

```diff
@@ -165,35 +172,49 @@ Test.prototype._end = function (err) {
 Test.prototype.exit = function () {
 	var self = this;
 
-	// calculate total time spent in test
-	this.duration = globals.now() - this._timeStart;
+	function checkAssertLength() {
```

Comment:
I'm not sure if this should be here, or it should be a method, or if it should be an external function which accepts `self`. Thoughts?